src/DynamoManipulation/MousePointManipulator.cs

```diff
@@ -36,8 +37,8 @@ protected override void AssignInputNodes()
         {
             //Default axes
             var axes = new Vector[] { Vector.XAxis(), Vector.YAxis(), Vector.ZAxis() };
-            //Holds manipulator axis and input node pair for each input port.
-            indexedAxisNodePairs = new Dictionary<int, Tuple<Vector, NodeModel>>(3);
+
+            indexedAxisNodePairs.Clear();
```

Comment:
just need to clear it where it was previously being recreated

Reply:
How does this fix the crash, not sure I understand?

Reply:
indexedAxisNodePairs will now be created when the constructor of the class is called.
So indexedAxisNodePairs will not be null anymore, only empty (which is properly handled)